Helper/Data.php

```diff
@@ -793,7 +793,7 @@ public function getOneClickPaymentMethods($customerId, $storeId, $grandTotal, $r
 
         $baCollection = $this->_billingAgreementCollectionFactory->create();
         $baCollection->addFieldToFilter('customer_id', $customerId);
-        $baCollection->addFieldToFilter('store_id', $storeId);
+        // $baCollection->addFieldToFilter('store_id', $storeId);
```

Comment:
instead of commenting out if won't be used in the future delete it I think

Reply:
Done